susemanager-utils/susemanager-sls/salt/certs/openeuler.sls

```diff
@@ -1 +1,24 @@
-redhat.sls
+{%- if grains['osrelease']|int == 6 %}
```

Comment:
I don't think we need such `if`s? When is `osrelease` going to be 6 for openeuler?

Reply:
This section indeed makes no sense. Removed.